lib/NormalModuleFactory.js

```diff
@@ -190,6 +265,10 @@ class NormalModuleFactory extends Tapable {
 				], (err, results) => {
 					if(err) return callback(err);
 					loaders = results[0].concat(loaders, results[1], results[2]);
+					/**
+					 * Why is this `process.nextTick` here? I would think intuitively that
+					 * this would slow down the build.
+					 */
 					process.nextTick(() => {
```

Comment:
Why is this `process.nextTick` here? I would think intuitively that this would slow down the build...

Reply:
Found the answer to this through happen stance. Most (all?) of a webpack build happens recursively passing callbacks. `process.nextTick` helps prevent running into max call stack errors when you recurse too deeply.